app/src/main/java/vn/sunasterisk/music_70/MainActivity.kt

```diff
@@ -80,6 +102,24 @@ class MainActivity : BaseActivity() {
         transaction.commit()
     }
 
+    fun initMiniPlaying() {
+        val fragmentTransaction = supportFragmentManager.beginTransaction()
+        val miniPlaying =
+            supportFragmentManager.findFragmentByTag(MiniPlayingFragment::class.java.name) as? MiniPlayingFragment
+        miniPlaying?.updateUI(mediaService.getCurrentTrack())?:run {
```

Comment:
Những chỗ logic phức tạp nên viết tường minh.
```suggestion
        miniPlaying?.let {
             updateUI(...)
             return
        }
        addFragment(...)
        
```